cmd/rosa/main.go

```diff
@@ -105,3 +113,32 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func versionCheck() {
+	rprtr := reporter.CreateReporterOrExit()
+
+	currVersion, err := gversion.NewVersion(info.Version)
+	if err != nil {
+		_ = rprtr.Errorf("There was a problem verifying the current version of rosa.")
```

Comment:
I'd recommend changing this to "Could not verify the current version of ROSA." or "Could not check ROSA version." making sure ROSA is capitalized